pandas/core/groupby/generic.py

```diff
@@ -1152,9 +1155,8 @@ def alt(obj):
 
     @property
     @doc(Series.plot.__doc__)
-    def plot(self):
-        result = GroupByPlot(self)
-        return result
+    def plot(self) -> GroupByPlot:
+        return GroupByPlot(self)
```

Comment:
When stepping through code with a debugger, you can at times get 4 function calls deep where at each level the code is `return foo(...)`. This requires you to back out 4 levels of the stack just to see what value it is you're returning, which can make debugging more tedious.

By having `result = foo(...)` first, you no longer need to do this.